conda.recipe/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: rise
+  version: 0.3.0
```

Comment:
This should be `3.x`... in fact, we probably need to add a `version` file somewhere in the future, but that's for later...